### PR TITLE
(DOCSP-26793): Expand metrics

### PR DESCRIPTION
**Pull request by cbush**
### Jira

- https://jira.mongodb.org/browse/DOCSP-26793

### Staged Changes

- https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/metrics-xpand/admin/api/v3/#tag/metrics
- https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/metrics-xpand/reference/metrics/
- 
### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [ ] Create Jira ticket for corresponding docs-realm update(s), if any
- [ ] Checked/updated Admin API
- [ ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:20]: should this be deleted?

File: source/openapi-admin-v3.yaml
Comment (mongodben) [inline source/openapi-admin-v3.yaml:2245]:
```suggestion
        Services Metrics Reference](https://mongodb.com/docs/atlas/app-services/reference/metrics).
```

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:65]:
```suggestion
    -  Failed authentication requests.
```

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:15]: nit: i don't think we refer to methods by their operationId elsewhere in the docs. 

i'd just put a plain-text description, smthn like
```rst 
:admin-api-endpoint:`Retrieve App Services metrics <tag/metrics/operation/adminGetMetrics>`
```

(also didn't know we had a `:admin-api-endpoint:` role)

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:18]: what's the purpose of the first sentence "The content on this page is provided for your information."?

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:41]: out of curiosity, do you know why this is the case?

seems weird to me that we're telling user that our metric is this good 5% or more of the time

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:56]: can you be more descriptive? 

is this the total number of subscriptions that were open over a given duration? 
including ones that were already open? excluding?

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:81]: how does this work with the time span in the query params? 

i/e number of users at beginning of time span could be different from the number at the end

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:167]: ditto https://github.com/mongodb/docs-app-services/pull/255/files#r1041320654

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:203]: should this just be bytes?

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:256]: again should just be bytes, not bytes/second?

**Review by mongodben — COMMENTED** [review]
back to you!

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:20]: Why?

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt]: Thanks, this is a search and replace flub. It's actually supposed to say Rate of ...

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:18]: Just a preamble to the next thing. But even that's questionable - eng says they might be private/subject to change but product would prefer they don't change.

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:15]: It's certainly inconsistent. Usually we refer to the category. I'm referring to a specific endpoint here, so I don't see the benefit of calling it something other than what it is. But I don't feel strongly.

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:41]: "Good" implies the metric is a value judgment and that higher is always better. It's neither. The 95th percentile scalar/sec type is used for things like response times where lower is actually better. It's just saying that MOST of the time, it's below X, but sometimes it's above X. I don't know the exact technical reason why it's presented this way.

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:85]: No. Note that the type is SCALAR_PER_SECOND, not MILLISECOND (or scalar). See the note about scalar per second. Basically this says how much time out of every second in the time window was spent computing. Really tried to keep this clean and clear but let me know if you have suggestions for how to clarify more, as I'm sure you're not the only one who would be left scratching your head.

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:146]: This is not supposed to be Total. This is a scalar per second value, which means it's actually way more confusing. Unlike the "milliseconds per second" construct above, this is minutes per second, which makes no sense...

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:154]: No. This comes from the source and from empirical testing. Almost everything has to do with rates as opposed to total values. But I can see I might need to clarify the phrases on the BYTES_PER_SECOND ones.

File: source/reference/metrics.txt
Comment (mongodben) [reply source/reference/metrics.txt:19]: i was told at some point to delete comments, but i have no personal issue w keeping. seems vaguely useful for future maintainers of the docs, assuming that spreadsheet is kept up to date.

File: source/reference/metrics.txt
Comment (mongodben) [reply source/reference/metrics.txt:15]: but the operationId isn't exposed in the spec (besides the URL path), so don't think it should be used here. recommending: 

```suggestion
:admin-api-endpoint:`Retrieve App Services metrics <tag/metrics/operation/adminGetMetrics>` endpoint.
```

File: source/reference/metrics.txt
Comment (mongodben) [reply source/reference/metrics.txt:40]: thanks for indulging my curiosity

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:55]: what do you mean by "time of reading"? is this not using the query params for `start` and `end`?

File: source/reference/metrics.txt
Comment (mongodben) [inline source/reference/metrics.txt:80]: ditto https://github.com/mongodb/docs-app-services/pull/255/files#r1042559324

File: source/reference/metrics.txt
Comment (mongodben) [reply source/reference/metrics.txt:84]: thanks for clarifying. i think the explanation above is good. it's a funky concept, and while it certainly could be expanded upon more, i don't know how necessary that would be for this fairly small (and presumably not highly trafficed) documentation page

**Review by mongodben — COMMENTED** [review]
back to you! looking good, just a couple of small clarification points before approval

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:15]: Oh, I see what you're saying. Got it.

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:19]: That was the idea. I usually say to delete things that are "commented out", not "comments".

File: source/reference/metrics.txt
Comment (cbush) [reply source/reference/metrics.txt:84]: We'll see if we get feedback on it. I could go with what I did for the really weird one "minutes per second". I said it's the number of minutes in the window divided by the number of seconds in that window. However, the problem(?) with that is that I'm then talking about "windows".

File: source/reference/metrics.txt
Comment (mongodben) [reply source/reference/metrics.txt:55]: > If the interval is like a day, is it the number of sessions open that day or is it a snapshot of the exact time of measurement? Not quite sure about this.

i think this is an important point to clarify.

> I think throwing in the word "reading" was a mistake. I can call it "measurement". Not sure if that alone would clarify anything but at least I wouldn't be using a new term. (FWIW, I chose the former because it sounded less stuffy but I changed my mind looking at it now.)

i'm partial to measurements..but maybe i'm just stuffy. defer to you on phrasing there. i think the above clarification point is what must be clarified for this PR.

**Review by mongodben — APPROVED** [review]
i think the addition clarifies it well. nice work here.

lgtm!

**Pull request merged**
